### PR TITLE
Chapter 10

### PR DESCRIPTION
**Pull request by huyennguyenthanh**
image

### CODE REVIEW

**Comment by huyennguyenthanh** [comment]
ready

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
   end
 
-  def show
-    @user = User.find_by id: params[:id]
-    return if @user
-
-    flash[:error] = t ".not_found"
-    redirect_to root_path
+  def index
+    @pagy, @users = pagy(User.all, page: params[:page], items: 10)
```
Comment (manhnt-0960) [inline app/controllers/users_controller.rb]: 10 đưa vào settings

File: app/views/users/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<%= form_for @user do |f| %>
+  <%= render "shared/error_messages" %>
+  <%= f.label :name %>
```
Comment (manhnt-0960) [inline app/views/users/_form.html.erb:3]: i18n

**Comment by huyennguyenthanh** [comment]
ready

**Comment by manhnt-0960** [comment]
merged

**Pull request merged**